test/files/secure-forward.conf

```diff
@@ -11,7 +11,7 @@
   <buffer>
     @type file
     path '/var/lib/fluentd/secureforwardlegacy'
-    queued_chunks_limit_size "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
+    queue_limit_length "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
```

Comment:
@periklis missed on here

Reply:
Actually this is a test configuration for a third-party managed fluentd receiver, which we only use on the corresponding E2E logfowarding test. See more [here](https://github.com/openshift/cluster-logging-operator/pull/479#discussion_r412995527)